README.md

```diff
@@ -111,6 +93,8 @@ Copyright (c) 2017 Calle Erlandsson, Anton Lindqvist & thoughtbot.
 [current]: https://github.com/calleerlandsson/pick/blob/master/CONTRIBUTING.md
 [debian]: https://packages.debian.org/stretch/pick
 [gentoo]: https://packages.gentoo.org/packages/sys-apps/pick
+[pick-examples]: https://mptre.github.io/pick.1#EXAMPLES
```

Comment:
Should this link point to the original repository?

Reply:
Not sure I follow. This is a reference to the HTML rendered manual.

Reply:
Right, I'm thinking it may cause confusion to users being redirected to another GitHub users website rather than the ones they started from. In this case Calle's project to your GitHub site.

It may be semantics, and doesn't really matter.

Reply:
Ah, we could stick it in the `gh-pages` branch for pick instead.

Reply:
Done